PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterCharacterScripts/Sound.rbxmx

```diff
@@ -83,7 +85,15 @@ local Sounds = {}
 do
 	local Figure = script.Parent.Parent
 	Head = Figure:WaitForChild("Head")
-	Humanoid = Figure:WaitForChild("Humanoid")
```

Comment:
sound has to be child of a part (Part object) in order to be 3D. this can be changed to just a general "IHasLocation" but wouldn't be any outward differences.